packages/pwa/app/commerce-api/auth.js

```diff
@@ -40,14 +41,26 @@ class Auth {
         this._onClient = typeof window !== 'undefined'
         this._pendingAuth = undefined
         this._customerId = undefined
-        this._authToken = this._onClient ? window.localStorage.getItem(tokenStorageKey) : undefined
-        this._refreshToken = this._onClient
-            ? window.localStorage.getItem(refreshTokenStorageKey)
-            : undefined
-        this._usid = this._onClient ? window.localStorage.getItem(usidStorageKey) : undefined
-        this._encUserId = this._onClient
-            ? window.localStorage.getItem(encUserIdStorageKey)
-            : undefined
+
+        this._oid = this._onClient ? window.localStorage.getItem(oidStorageKey) : undefined
+
+        let configOid = api._config.parameters.organizationId
+        if (this._oid != configOid) {
```

Comment:
It is a good practise to use strict equality check unless we have a reason to do otherwise 
```suggestion
        if (this._oid !== configOid) {
```